packages/api/bin/serveUtils.js

```diff
@@ -1,72 +1,183 @@
 'use strict';
 
 const indexer = require('@cumulus/es-client/indexer');
+const {
+  GranulesExecutionsPgModel,
+  RulePgModel,
+  GranulePgModel,
+  PdrPgModel,
+  ExecutionPgModel,
+  CollectionPgModel,
+  ProviderPgModel,
+  translateApiCollectionToPostgresCollection,
+  translateApiProviderToPostgresProvider,
+  translateApiExecutionToPostgresExecution,
+  translateApiGranuleToPostgresGranule,
+  translateApiPdrToPostgresPdr,
+  translateApiRuleToPostgresRule,
+  getKnexClient,
+  localStackConnectionEnv,
+  envParams,
+  createTestDatabase,
+  migrationDir,
+} = require('@cumulus/db');
+const { log } = require('console');
 const models = require('../models');
 const { getESClientAndIndex } = require('./local-test-defaults');
+const {
+  erasePostgresTables,
+} = require('./serve');
+
+async function resetPostgresDb() {
+  const knexAdmin = await getKnexClient({ env: localStackConnectionEnv });
+  const knex = await getKnexClient({
+    env: {
+      ...envParams,
+      ...localStackConnectionEnv,
+      migrationDir,
+    },
+  });
+
+  try {
+    await createTestDatabase(knexAdmin, 'postgres', localStackConnectionEnv.PG_USER);
+  } catch (error) {
+    log(`Skipping Postgres DB creation because ${error}`);
+  }
+
+  await knex.migrate.latest();
+
+  await erasePostgresTables(knex);
+}
```

Comment:
Created a `resetPostgresDb` because between each suite of integration tests we reset the dynamoDb tables and ES without restarting the API. This method simply empties each table and ensures the latest schema is applied. It will create the database if it doesn't exist (ie first run).